src/sdkconfig.h

```diff
@@ -0,0 +1,371 @@
+/*
```

Comment:
is this file necessary since its autogenerated?

Reply:
see comment above

Reply:
this file is generated by `idf.py menuconfig` or similar, but platformio doesn't seem to support it and recommends to modify `sdkconfig.h` directly.
https://github.com/platformio/platform-espressif32/issues/20
https://community.platformio.org/t/esp32-idf-menuconfig-platofrmio-dragging-their-feet/6268
should we also install idf in the container and use their menuconfig or wait this out?

Reply:
I would say we wait this out as pio allows us to modify the ```sdkconfig.h``` file there is no need to increase complexity

Reply:
ok